jdbc/src/java/org/apache/hive/jdbc/HiveStatement.java

```diff
@@ -360,6 +360,9 @@ TGetOperationStatusResp waitForOperationToComplete() throws SQLException {
     // Poll on the operation status, till the operation is complete
     do {
       try {
+        if (Thread.currentThread().isInterrupted()) {
+          throw new SQLException("Interrupted while polling on the operation status", "70100");
```

Comment:
I think this error message and code should be placed in the `ErrorMsg` class

Reply:
Done, thank you very much! @kgyrtkirk